test/utils.cpp

```diff
@@ -119,6 +119,11 @@ TEST_CASE("Tokenizers behave correctly", "[utils]") {
 	}
 }
 
+TEST_CASE("utils::consolidate_whitespace bahaves correctly", "[utils]") {
```

Comment:
In new tests, I'm moving away from "behaves correctly", using more descriptive things like "replaces multiple consecutive whitespace with single space". Can you please update this?